coding-standards/html-css/README.md

```diff
@@ -175,8 +205,8 @@ Some examples of where comments are a necessity:
 
 ### Assets
 
-- Only use open source fonts or fonts with a license
-- Preload webfonts
+- Only use open source fonts or fonts with a obtained license
```

Comment:
Typo: _an obtained license_

Reply:
Refactoring of sentences takes its toll...